source/includes/administration/_discounts.md

```diff
@@ -0,0 +1,63 @@
+## Discounts
+
+<!-------------------- LIST DISCOUNTS -------------------->
+### List discounts
+
+`GET /applied_pricings/:applied_pricing_id/discounts?type=:type`
+
+Retrieves the list of discounts associated with an applied pricing.
+
+```shell
+# Retrieve applied pricing list
+curl "https://cloudmc_endpoint/rest/applied_pricings/efd32752-c6f2-45cf-b494-cc6be8a45845/discounts?type=PERCENTAGE" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "applyToNewCustomersOnly": true,
+      "discountedProducts": {},
+      "durationMonths": 22354,
+      "type": "PERCENTAGE",
+      "packageDiscount": 20.0,
+      "discountScope": "ALL_PRODUCTS",
+      "isDeactivated": true,
+      "discountedCategories": {},
+      "name": {
+        "en": "Summer Discount",
+        "fr": "Réduction Estival"
+      },
+      "id": "18db7bc6-8be1-48bb-bab1-77a7d696fa3b",
+      "appliedPricing": {
+        "id": "efd32752-c6f2-45cf-b494-cc6be8a45845"
+      },
+      "startDate": "2021-07-20T15:57:16.132Z",
+      "status": "CURRENT"
+    }
+  ]
+}
+```
+Optional Query Parameters | &nbsp;
+---------- | -----------
+`type`<br/>*enum* | The type of the discount. It could be either "PERCENTAGE" or "CREDIT".
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the discount.
+`discountedProducts`<br/>*Map[UUID, BigDecimal]* | A mapping of the desired priced product IDs and discounts. To provide a 20 percent discount, set the discount value to 20.
+`durationMonths`<br/>*integer* | Duration of the discount. If not provided the discount will last indefinitely once applied to a customer, or until credit values are reached.
```

Comment:
maybe we change it to 

"Duration of the discount once it has been applied to a customer. If not provided the discount will last indefinitely, or until credit values are reached."

just so we are super clear with the duration and cutoff date